javascript/lastfm.js

```diff
@@ -243,17 +243,17 @@ async function saveToStorage() {
   try {
     const apiKey = apiKeyInput.value;
     if (!apiKey) {
-      await somethingWentWrong("API key is required");
-      throw new Error("API key is required");
+      somethingWentWrong('API key is required!');
+      resetToFirstState();
```

Comment:
Why ``resetToFirstState()`` again? As I said, you are just wasting computational power and bandwidth loading the page again lmao